CHANGES.rst

```diff
@@ -568,6 +568,8 @@ astropy.wcs
 - Fix a bug that caused APE 14 WCS transformations for FITS WCS with ZOPT, BETA,
   VELO, VOPT, or VRAD CTYPE to not work correctly. [#11781]
 
+- Enabled ``SlicedLowLevelWCS.pixel_to_world_values`` to handle slices including non- ``int``
+  integers, e.g. ``numpy.int64``.
```

Comment:
Astropy now uses towncrier. You should put this in `docs/changes/wcs/11980.bugfix.rst`.

Reply:
Please squash out the changes to CHANGES.rst too.